src/placeos-rest-api/utilities/current-user.cr

```diff
@@ -94,6 +97,22 @@ module PlaceOS::Api
       token.is_support? || token.is_admin?
     end
 
+    def can_scope_read(scope_name : String)
```

Comment:
With the above change we can this to be
`can_scope_read(*scope_names)`
where we return `true` if the user has access to any of these scopes

Reply:
Do you mean `all` of the scopes?